### ISSUE TITLE
Keyboard Accessibility

### ISSUE THREAD

**Issue by aferreira-deo** [ivylaboratory/angular-carousel#144]
The carousel next and previous buttons should be usable with a keyboard only in order to match the WCAG accessibility standards, more precisely WCAG 2.1.1 https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html.

In order to do this you could code the next/previous buttons as buttons instead of divs. Alternatively if you still want to keep them as divs then you can add a tabindex=0, click and keyboard handlers for the enter/space keys, and focus  styling for when the keyboard focuses the buttons.

Best regards
